src/client/session/mod.rs

```diff
@@ -561,6 +566,107 @@ impl ClientSession {
         }
     }
 
+    /// Runs a callback inside a transaction.  Transient transaction errors will cause the callback
```

Comment:
Can we add a bit more detail here explaining that the transaction will be started and committed/aborted within this method?

Reply:
Updated.